docs/user/dashboard/lens-advanced.asciidoc

```diff
@@ -57,13 +57,15 @@ image::images/lens_clickAndDragZoom_7.16.gif[Cursor clicking and dragging across
 
 . In the layer pane, click *Count of Records*.
 
-.. In the *Name* field, enter `Number of orders`.
-
-.. Click *Add advanced options > Normalize by unit*. 
+.. Click *Advanced*. 
 
 .. From the *Normalize by unit* dropdown, select *per hour*, then click *Close*.
 +
-*Normalize unit* converts *Average sales per 12 hours* into *Average sales per 12 hours (per hour)* by dividing the number of hours.
+*Normalize unit* converts `Average sales per 12 hours` into `Average sales per 12 hours (per hour)` by dividing the number of hours.
```

Comment:
Can you provide a bit more explanation about what is being divided? I don't understand the difference between the two labels.  Maybe just show the new label?

Reply:
Thanks for flagging this @gchaps , we should definitely change this. "Normalize by unit" is only available for summable functions like count and sum, not for averages as it doesn't make any sense for these.

The idea is that the data is divided into intervals automatically, e.g. "Sum of sales per 12 hours", which changes based on the chosen time range - e.g. when looking at the full last year it becomes "Sum of sales per 24 hours" automatically to avoid transferring too much data. To make sure the value is always shown for the same time unit to stay comparable, "Normalize by unit (hour)" can be used to always show "Sum of sales per hour", no matter the actual underlying interval (which is done by dividing the value by the factor between the date histogram granularity and the desired unit to show the values - in this case 24 hours to 1 hour means dividing by 24).